spec/views/medicaid/income_other_income_type/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+describe "medicaid/income_other_income_type/edit.html.erb" do
+  before do
+    controller.singleton_class.class_eval do
+      def current_path
+        "/steps/medicaid/income-other-income-type"
+      end
+
+      def current_member
+        Member.last
+      end
+
+      def current_application
+        current_member.benefit_application
+      end
+
+      helper_method :current_path, :current_application, :current_member
```

Comment:
why not make these private methods instead?

Reply:
I'm not sure I understand. As opposed to / instead of ...?